l10n_be_mis_reports/__manifest__.py

```diff
@@ -13,9 +13,32 @@
     "depends": ["mis_builder", "l10n_be"],  # OCA/account-financial-reporting
     "data": [
         "data/mis_report_styles.xml",
-        "data/mis_report_pl.xml",
-        "data/mis_report_bs.xml",
+        "data/mis_report_bs_m01-f.xml",
+        "data/mis_report_pl_m01-f.xml",
+        "data/mis_report_bs_m02-f.xml",
+        "data/mis_report_pl_m02-f.xml",
+        "data/mis_report_bs_m04-f.xml",
+        "data/mis_report_pl_m04-f.xml",
+        "data/mis_report_bs_m05-f.xml",
+        "data/mis_report_pl_m05-f.xml",
+        "data/mis_report_bs_m07-f.xml",
+        "data/mis_report_pl_m07-f.xml",
+        "data/mis_report_bs_m08-f.xml",
+        "data/mis_report_pl_m08-f.xml",
+        "data/mis_report_bs_m81-f.xml",
+        "data/mis_report_pl_m81-f.xml",
+        "data/mis_report_bs_m82-f.xml",
+        "data/mis_report_pl_m82-f.xml",
+        "data/mis_report_bs_m87-f.xml",
+        "data/mis_report_pl_m87-f.xml",
+        "data/mis_report_bs_deprecated.xml",
+        "data/mis_report_pl_deprecated.xml",
         "data/mis_report_vat.xml",
     ],
     "installable": True,
+    "external_dependencies": {
+        "python": [
+            "click",
```

Comment:
I'd remove this, since it is only necessary for generating the template. So let's just documented in the README of the script directory.